### ISSUE TITLE
Add support for tvOS (`aarch64-apple-tvos`) to Apple bindings

### ISSUE THREAD

**Issue by lopopolo** [strawlab/iana-time-zone#71]
It looks like `core-foundation-sys` supports tvOS. I think this should be an easy fix apart from adding the `cargo check` to CI.

**Comment by Kijewski** [comment]
Yes, I think it's fine to enable tvOS support, even if untested. I did the same for solaris in #55, because it *should* work the same as in illumos.
